integrate.js

```diff
@@ -81,6 +81,50 @@
     this.update()
   }
 
+  WebApp._getShuffle = function () {
+    var elm = this.getButtons().shuffle
+    return elm ? elm.classList.contains('player-controls__btn_on') : null
+  }
+
+  WebApp._getRepeat = function () {
+    var elm = this.getButtons().repeat
+    if (!elm) {
+      return null
+    }
+
+    if (elm.classList.contains('player-controls__btn_repeat_state1')) {
+      return Nuvola.PlayerRepeat.PLAYLIST
+    } else if (elm.classList.contains('player-controls__btn_repeat_state2')) {
+      return Nuvola.PlayerRepeat.TRACK
+    } else {
+     return Nuvola.PlayerRepeat.NONE 
+    }
+  }
+
+  WebApp._setRepeat = function (repeat) {
+    /*
+     * There are mismatch between Nuvola's states order and Yandex.Music states order
+     * and this map defines how much repeat button should be clicked, to change repeat state from current
+     * to requested
+     */
+    var clicksMap = {
+      "00": 0,
+      "01": 2,
+      "02": 1,
+      "10": 1,
+      "11": 0,
+      "12": 2,
+      "20": 2,
+      "21": 1,
+      "22": 0
+    };
```

Comment:
This is not in Standard JS code style. Could you run `standard --fix` from inside the Nuvola ASK sandbox? In future, this will be checked with Continuous Integration tiliado/nuvolaruntime#420